src/screens/Notifications/components/NotificationCard/NotificationCard.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {View, Text, Image} from 'react-native';
+
+import generateName from 'sillyname';
```

Comment:
i don't like this way for generation, we just may use https://randomuser.me/ . We must remove it when we add a actual data

Reply:
they are the same, and we will get rid of this dummy images soon. We will have fixtures on a server side to generate images there. Let's keep 'sillyname' module instead, at leas for now.